### ISSUE TITLE
Passing false in ShipmentServiceOptions setters won't do anything

### ISSUE THREAD

**Issue by compadvance** [gabrielbull/php-ups-api#133]
There is serious issue with ShipmentServiceOptions. 

See example, this:

```
$sso = new ShipmentServiceOptions();
$sso->setSaturdayDelivery(false);
```

will make Saturday delivery because $SaturdayDelivery will be set

```
public function setSaturdayDelivery($SaturdayDelivery)
{
    $this->SaturdayDelivery = $SaturdayDelivery;
    return $this;
}
public function toNode(DOMDocument $document = null)
{
    ...
    if (isset($this->SaturdayDelivery)) {
        $node->appendChild($document->createElement('SaturdayDelivery'));
    }
    ...
}
```

This apply to other bool options in class

UPS documentation says

> Saturday delivery indicator. The presence indicates Saturday delivery, the absence indicates not Saturday delivery.

**Comment by stefandoorn** [comment]
True, but if you leave out the $sso->setSaturdayDelivery(false) the problem does not exist. Would that work in your situation? They also leave it out in the response if not set as far as I know, so we are compatible with their XML responses.

So only use 'setSaturdayDelivery' when you want to use Saturday Delivery in this case.

**Comment by compadvance** [comment]
So method shouldn't have any argument.

**Comment by gabrielbull** [comment]
It would be better if it could be set and unset, so the method should retain the argument, but setting saturdayDelivery to false should remove the option form the UPS XML call. Can you make a PR @compadvance ?

**Comment by stefandoorn** [comment]
Basically setting it to `null` as argument, would remove it from the call. Anything else will add it. I can imagine people would expect that false should also be able to unset it. A PR would be great regarding that, indeed. Maybe just the isset() check should be adjusted to isset && !empty or something similar.

**Comment by franklininnocent** [comment]
$service = new \Ups\Entity\ShipmentServiceOptions();
        $service->setSaturdayDelivery(false);
        $shipment->setShipmentServiceOptions($service);

This code is not working. getting error "Failure: No matching Rate and Transit times available. (111562)"
